src/analysis/inner_product_space/finite_dimensional.lean

```diff
@@ -0,0 +1,354 @@
+/-
+Copyright (c) 2023 Monica Omar. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Monica Omar
+-/
+import linear_algebra.invariant_submodule
+import analysis.inner_product_space.adjoint
+import analysis.inner_product_space.spectrum
+
+/-!
+# Finite-dimensional inner product spaces
+
+In this file, we prove some results in finite-dimensional inner product spaces.
+
+## Notation
+
+This file uses the local notation `P _` for `orthogonal_projection _`
+and `↥P _` for the extended orthogonal projection `V →L[ℂ] V`.
+
+We let `V` be an inner product space over `ℂ`.
+-/
+
+variables {V : Type*} [inner_product_space ℂ V]
```

Comment:
As far as I can tell, you don't use anything complex-specific in (the beginning of) this file, so you should probably just use 
```suggestion
variables {𝕜 V : Type*} [is_R_or_C 𝕜] [inner_product_space 𝕜 V]
```